include/inviwo/core/util/volumeramutils.h

```diff
@@ -30,8 +30,12 @@
 #pragma once
 
 #include <inviwo/core/common/inviwocoredefine.h>
-#include <inviwo/core/datastructures/volume/volumeram.h>
 #include <inviwo/core/common/inviwoapplication.h>
```

Comment:
remove?

Reply:
required in `forEachVoxel()`
https://github.com/inviwo/inviwo/blob/bbef904fa5898e276ca5b13ba0f6c2413c3779bc/include/inviwo/core/util/volumeramutils.h#L52-L55

Reply:
I meant the inviwoapplication.h include